README.md

```diff
@@ -59,16 +62,44 @@ with open('my_experiment_config.yaml') as f:
 with open('my_database_creds') as f:
 	db_connection_string = yaml.load(f)['db_connection_string']
 
+logging.basicConfig(level=logging.INFO)
+
 experiment = SingleThreadedExperiment(
 	config=experiment_config,
 	db_engine=sqlalchemy.create_engine(db_connection_string),
 	model_storage_class=FSModelStorageEngine,
 	project_path='/home/research/myproject'
 )
+```
+
+### Validating an Experiment
+
+Configuring an experiment is very complicated, and running an experiment can take a long time as data scales up. If there are any misconfigured values, it's going to help out a lot to figure out what they are before we run the Experiment. So we recommend running the `.validate()` method on the Experiment first. If any problems are detectable in your Experiment, either in configuration or the database tables referenced by it, this method will throw an exception. For instance, if I refer to the 'cat_complaints' table in a feature aggregation but it doesn't exist, I'll see something like this:
+
 
-experiment.run()
 ```
+experiment.validate()
+
+(Pdb) experiment.validate()
+*** ValueError: from_obj query does not run. 
+from_obj: "cat_complaints"
+Full error: (psycopg2.ProgrammingError) relation "cat_complaints" does not exist
+LINE 1: explain select * from cat_complaints
+                              ^
+ [SQL: 'explain select * from cat_complaints']
+```
+
+If nothing is output, it means that we can't find anything wrong with the Experiment so far, and it should be ready to run.
```

Comment:
Just from a user-friendliness perspective, might actually be nice to output a message saying that no validation errors were found.